src/renderers/dom/client/wrappers/ReactDOMInput.js

```diff
@@ -81,7 +82,7 @@ var ReactDOMInput = {
       onChange: inst._wrapperState.onChange,
     });
 
-    return nativeProps;
+    return DisabledInputUtils.getNativeProps(inst, nativeProps);
```

Comment:
Why is `DisabledInputUtils.getNativeProps` called last here unlike `ReactDOMSelect` and all other components where it is called inline in the `assign` call?